### ISSUE TITLE
Problem Installing pybgpstream

### ISSUE THREAD

**Issue by wicaksana** [CAIDA/bgpstream#21]
I tried to install pybgpstream on a virtual env of Python 3.5 on Ubuntu 16.04 64-bit. I have successfully installed BGPStream. When I installed using pip (`pip install pybgpstream`), I got the following error:

> (omitted) 
> src/_pybgpstream_bgpstream.c:439:37: note: (near initialization for ‘BGPStreamType’)
>     src/_pybgpstream_bgpstream.c:481:15: warning: function declaration isn’t a prototype [-Wstrict-prototypes]
>      PyTypeObject *_pybgpstream_bgpstream_get_BGPStreamType()
>                    ^
>     error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
>     
> 
> Command "/home/arif/Github/anycast-dns-monitoring-framework/env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-zh56p_38/pybgpstream/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-41tnpipb-record/install-record.txt --single-version-externally-managed --compile --install-headers /home/arif/Github/anycast-dns-monitoring-framework/env/include/site/python3.5/pybgpstream" failed with error code 1 in /tmp/pip-build-zh56p_38/pybgpstream/

From my search in google, similar problem is usually caused by the absence of `python-dev`. However, I have already installed the latest `python3-dev` (Python 3.5).

What could be the possible cause of my problem?  

Update:
I am able to install it on Python 2.7, but I would like to use Python 3.5 if it is possible.

**Comment by alistairking** [comment]
This is because the currently released version of PyBGPStream does not support Python 3.
This has been fixed by PR #11, so if you can, please install pybgpstream from the master branch of this repo, otherwise, we expect to have make a new release in the next few weeks.

**Comment by saqib772** [comment]
Hi, I'm Getting the following error when I try to install the pybgpstream on windows and on UBUNTU. Tried everything no solution.

```
Collecting pybgpstream
  Using cached pybgpstream-2.0.2.tar.gz (11 kB)
  Preparing metadata (setup.py) ... done
Requirement already satisfied: python-dateutil in /usr/local/lib/python3.10/dist-packages (from pybgpstream) (2.8.2)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil->pybgpstream) (1.16.0)
Building wheels for collected packages: pybgpstream
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Building wheel for pybgpstream (setup.py) ... error
  ERROR: Failed building wheel for pybgpstream
  Running setup.py clean for pybgpstream
Failed to build pybgpstream
ERROR: Could not build wheels for pybgpstream, which is required to install pyproject.toml-based projects
```
